tests/pyupgrade_test.py

```diff
@@ -1448,6 +1448,30 @@ def test_fix_native_literals(s, expected):
     assert _fix_py3_plus(s) == expected
 
 
+@pytest.mark.parametrize(
+    ('s', 'expected'),
+    (
+        ('"asd".encode("utf-8")', 'b"asd"'),
+        ('"asd".encode("UTF-8")', 'b"asd"'),
+        ('"asd".encode(encoding="utf-8")', 'b"asd"'),
+        ('"asd".encode(encoding="UTF-8")', 'b"asd"'),
+
+        ('"☕️".encode("utf-8")', '"☕️".encode()'),
+        ('"☕️".encode("UTF-8")', '"☕️".encode()'),
+        ('"☕️".encode(encoding="utf-8")', '"☕️".encode()'),
+        ('"☕️".encode(encoding="UTF-8")', '"☕️".encode()'),
+
+        # the current version only works for string literals
+        ('x="asd"\nx.encode("utf-8")', 'x="asd"\nx.encode("utf-8")'),
```

Comment:
the general trend I've done is to add a separate parametrized `_noop` test so it's easier to see

Reply:
sure